docs/example.md

```diff
@@ -185,7 +245,7 @@ For JDBC, search for <span style= "font-family: courier new">Connection URL</spa
 JDBCProtocol Connection URL: jdbc:postgresql://postgres:5342/TestV1?user=test&password=********
```

Comment:
```suggestion
JDBCProtocol Connection URL: jdbc:postgresql://postgres:5342/TestV1
```
This example pre-dated the propertiesBuilder.  user/pass are no longer passed on the url.

Reply:
Fixed